app/models/event.rb

```diff
@@ -26,6 +26,12 @@ class Event < ActiveRecord::Base
 
   acts_as_sequenced scope: :discussion_id, column: :sequence_id, skip: lambda {|e| e.discussion.nil? || e.discussion_id.nil? }
 
+  def active_model_serializer
+    "Events::#{eventable.class.to_s.split('::').last}Serializer".constantize
```

Comment:
This is a hook into active model serializers, which lets the model say how it should be serialized.

It looks at the eventable class to determine if there should be any special serializer behaviour (like, if it's a motion eventable, we need say 'root: :proposals' for example), otherwise go with the Events::BaseSerializer